source/evidencevariable/structuredefinition-EvidenceVariable.xml

```diff
@@ -568,5 +568,207 @@
 				<targetProfile value="http://hl7.org/fhir/StructureDefinition/Group"/>
 			</type>
 		</element>
+		<element id="EvidenceVariable.conditional">
+			<path value="EvidenceVariable.conditional"/>
+			<short value="Context in which the variable is observed"/>
+			<definition value="The context, situation, or parameters within which the variable is observed."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="Expression"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.classifier">
+			<path value="EvidenceVariable.classifier"/>
+			<short value="Classification"/>
+			<definition value="Classification of the variable."/>
+			<min value="0"/>
+			<max value="*"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.datatype">
+			<path value="EvidenceVariable.datatype"/>
+			<short value="Type of data used to express value of the variable"/>
+			<definition value="The type of data used to express values of the variable."/>
+			<min value="0"/>
+			<max value="*"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.timing">
+			<path value="EvidenceVariable.timing"/>
+			<short value="When the variable is observed"/>
+			<definition value="When the variable is observed in relation to a reference point in time defined by context or event."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="RelativeTime"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.period">
+			<path value="EvidenceVariable.period"/>
+			<short value="Date(s) and time(s) when the variable is observed"/>
+			<definition value="When the variable is observed in relation to calendar dates and times."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="Period"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.constraint">
+			<path value="EvidenceVariable.constraint"/>
+			<short value="Limit on acceptability of data value"/>
+			<definition value="Limit on acceptability of data used to express values of the variable."/>
+			<min value="0"/>
+			<max value="*"/>
+			<type>
+				<code value="BackboneElement"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.constraint.conditional">
+			<path value="EvidenceVariable.constraint.conditional"/>
+			<short value="When this constraint applies"/>
+			<definition value="When this constraint applies."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
```

Comment:
The ticket indicates contraint.conditional should have an Example binding?

Reply:
Added the binding.